tools/docker/README.md

```diff
@@ -23,3 +23,19 @@ docker build \
   -f tools/docker/Dockerfile \
   .
 ```
+
+Simple build instructions for newbies.  The sysctl is important for preventing errors when the tests run.
+
+```bash
+git clone git@github.com:vectorizedio/redpanda
+cd redpanda/
+
+# pull toolchain image
+docker pull vectorized/redpanda-toolchain
+
+# set fs.max-aio-nr an appropriate value
+sudo sysctl -w fs.aio-max-nr=10485760
```

Comment:
looks like this is the command that is new with respect to the what the README already had. so i think this could instead add this command to the instructions above, and also the note on sysctl being important for unit tests.

Reply:
@ivotron does the sysctl take effect w/o the --privileged flag?

Reply:
running `sysctl` inside a container can be done by using the `--sysctl` flag of `docker run` but this doesn't work for all sysctls (workarounds exists by mounting `/proc`). in this scenario though, `sysctl` is executed on the host